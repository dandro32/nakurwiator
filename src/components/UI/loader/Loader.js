import React from 'react';
import LoadingBar from 'react-redux-loading-bar';
const Loader = () => {
  return ( 
    <LoadingBar className="progress-bar" />
  );
};
export default Loader;