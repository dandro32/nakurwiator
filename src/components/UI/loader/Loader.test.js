import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Loader from './Loader';

describe('Loader component tests', ()=> {
  it('should render loader component', () => {
    const shallowRenderer = new ShallowRenderer();
    shallowRenderer.render(<Loader />);
    expect(shallowRenderer.getRenderOutput()).toMatchSnapshot();
  });

});