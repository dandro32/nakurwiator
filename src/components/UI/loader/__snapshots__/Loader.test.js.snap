// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component tests should render loader component 1`] = `
<Connect(LoadingBar)
  className="progress-bar"
/>
`;
